test/test_property.py

```diff
@@ -142,6 +142,11 @@ def test_text_property_setting_list(self):
         tp.value = expected
         self.assertEqual(tp.value, [])
 
+    def test_owned_object_singleton(self):
```

Comment:
Can a test be added to ensure that OwnedObject attributes with cardinality greater than 1 return a list of OwnedObjects? That way we won't accidentally break the backward-compatible behavior.

Reply:
Done